### PR TITLE
container: Enable compression by default

### PR DESCRIPTION
**Pull request by cgwalters**
I'd noticed that some of our images were large but hadn't looked
closely.  I now really wish I had...

I think this regressed quite a while ago during a refactoring.

### CODE REVIEW

**Review by jmarrero — APPROVED** [review]
lgtm

**Comment by cgwalters** [comment]
Note I plan to circle back here and use the infrastructure in https://github.com/ostreedev/ostree-rs-ext/pull/327 to add a unit test for this.

File: lib/src/container/encapsulate.rs [hunk 1]
```diff
@@ -283,8 +283,8 @@ async fn build_impl(
 /// Options controlling commit export into OCI
 #[derive(Debug, Default)]
 pub struct ExportOpts {
-    /// If true, perform gzip compression of the tar layers.
-    pub compress: bool,
+    /// If false, do not perform gzip compression of the tar layers.
+    pub skip_compression: bool,
```
Comment (jlebon) [inline lib/src/container/encapsulate.rs:287]: This looks odd. Shouldn't it be "If false, perform gzip compression of the tar layers" ? (Or maybe reword as "If true, skip gzip compression of the tar layers".) Another way to see this I think is we added one negation in the field name, but two negations in the docstring.

File: lib/src/container/encapsulate.rs [hunk 2]
```diff
@@ -237,7 +237,7 @@ async fn build_impl(
 ) -> Result<String> {
     let mut opts = opts.unwrap_or_default();
     if dest.transport == Transport::ContainerStorage {
-        opts.compress = false;
+        opts.skip_compression = false;
```
Comment (jlebon) [inline lib/src/container/encapsulate.rs:240]: And this should be `true`, no?

**Comment by cgwalters** [comment]
Thanks, followup in https://github.com/ostreedev/ostree-rs-ext/pull/330
